src/config.c

```diff
@@ -27,13 +27,15 @@ int ReadConfig(RedisModuleString **argv, int argc) {
             return TSDB_ERROR;
         }
 
-        printf("loaded default compation policy: %s\n\r", policy_cstr);
+        printf("loaded default compaction policy: %s\n\r", policy_cstr);
         TSGlobalConfig.hasGlobalConfig = TRUE;
     }
 
     if (argc > 1 && RMUtil_ArgIndex("RETENTION_POLICY", argv, argc) >= 0) {
```

Comment:
There's a much better argument parsing system that we have in RediSearch that replaces RMUtil_ParseArgs.. Might be useful to look at this later on. Also, suggest using the RETENTION_POLICY as a constant (e.g. macro)